pynet/datasets/biomede.py

```diff
@@ -0,0 +1,72 @@
+from collections import namedtuple
+import os
+import logging
+import numpy as np
+import pandas as pd
+import nibabel as nib
+import progressbar
+from pynet.datasets import Fetchers
+
+# Global parameters
+MODALITIES = ("t1", "t1ce", "t2", "flair")
+Item = namedtuple("Item", ["input_path", "output_path", "metadata_path"])
+logger = logging.getLogger("pynet")
+
+
+@Fetchers.register
+def fetch_biomede(datasetdir):
+    """ Fetch/prepare the Brats dataset for pynet.
+
+    Parameters
+    ----------
+    datasetdir: str
+        the dataset destination folder.
+
+    Returns
+    -------
+    item: namedtuple
+        a named tuple containing 'input_path', 'output_path', and
+        'metadata_path'.
+    """
+    logger.info("Loading biomede dataset.")
+
+    def _crop(arr):
+        return arr[45: 195, 30: 220, :145]
+
+    def _norm(arr, mask=None):
+        logical_mask = (arr != 0) if mask is None else mask
+        mean = arr[logical_mask].mean()
+        std = arr[logical_mask].std()
+        return ((arr - mean) / std).astype(np.single)
+
+    traindir = os.path.join(datasetdir, "biomede")
+    mapping_path = os.path.join(traindir, "name_mapping.csv")
+    if not os.path.isfile(mapping_path):
+        raise ValueError(
+            "You must first prepare biomede dataset")
```

Comment:
Just specify that this dataset is not public and that you need to request specific access rights.

Reply:
The CI is in error.
The code does not respect the coding style.
The explicit error can be found on Travis-CI server: https://travis-ci.org/github/neurospin/pynet/builds/694555367?utm_source=github_status&utm_medium=notification